content/developer/vanilla-cli/addon-utilities.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Addon Utilities
+tags:
+- Developers
+- CLI
+- addon-doctor
+- addon-json
+- addon-cache
+category: developer
+menu:
+  developer:
+    parent: cli
+---
+
+These commands all make managing addons a little bit easier. These commands currently require a functioning Vanilla installation >= `2.4.201`. They only work on addons currently installed in that Vanilla installation and function by using Vanilla's built in addon manager to do the heavy lifting. As a result, these commands require you to point them to the vanilla directory with the `--vanillasrc` parameter.
```

Comment:
Why not use versioning here? We've got support for both the page and sub sections.